src/firefox/remote.js

```diff
@@ -86,27 +86,50 @@ export class RemoteFirefox {
     });
   }
 
+  getAddonsActor(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      // getRoot should work since Firefox 55 (bug 1352157).
+      this.client.request('getRoot', (err, actors) => {
+        if (!err) {
+          if (actors.addonsActor) {
+            resolve(actors.addonsActor);
+          } else {
+            reject(new RemoteTempInstallNotSupported(
+              'This version of Firefox does not provide an add-ons actor for ' +
+              'remote installation.'));
```

Comment:
If this is an unexpected condition as I think it is, this error message sounds a bit misleading for that purpose.

Should we at least suggest to the users that are experiencing this error to report an issue in the web-ext repo?

(and maybe use just `WebExtError` or a new subclass, instead of `RemoteTempInstallNotSupported`).